bin/dwibiascorrect

```diff
@@ -118,7 +118,7 @@ elif app.args.ants:
   run.command('mrconvert mean_bzero.mif mean_bzero.nii -stride +1,+2,+3')
   run.command('mrconvert mask.mif mask.nii -stride +1,+2,+3')
   bias_path = 'bias.nii'
-  run.command('N4BiasFieldCorrection -d 3 -i mean_bzero.nii -w mask.nii -o [corrected.nii,' + bias_path + '] -s 2 -b [150] -c [200x200,0.0]')
+  run.command('N4BiasFieldCorrection -d 3 -i mean_bzero.nii -w mask.nii -o [corrected.nii,' + bias_path + '] -b [150,3] -c [1000x1000,0.0]')
```

Comment:
Takes a tiny bit longer, yet still fast... but greatly improves suppression of non-brain bits in the lower frontal part of the head (nasal cavity and related stuff), which makes life much less difficult for `dwi2mask`.